.github/pull_request_template.md

```diff
@@ -1,17 +1,17 @@
-What is the change?
+# What is the change?
 
 
-What does it fix/add?
+# What does it fix/add?
 
 
-How was it tested?
+# How was it tested?
 
 
-Submissions guide:
+# Submissions guide:
```

Comment:
Scale it to `##` from `#`

Reply:
@Neilblaze Should this heading be `##` or `###`. Just confirming that this isn't a typo since all other headings are `###`.

Reply:
no it's `##`